data/operators/amenity/charging_station.json

```diff
@@ -788,6 +788,14 @@
         "operator": "Daimler AG"
       }
     },
+    {
+      "displayName": "da emobil GmbH & Co KG",
+      "locationSet": {"include": ["at", "de", "ch", "gb", "ie", "it", "fr", "hu", "dk"]},
+      "tags": {
+        "amenity": "charging_station",
+        "operator": "da emobil GmbH & Co KG"
```

Comment:
same here

Reply:
Here you can use whatever `operator` value the local community prefers.  Most local communities don't like to see the "GmBH & Co KG" stuff, but maybe yours does?

The important thing for this to be converted to a preset is to add a `operator:wikidata` tag  (which this PR doesn't have).